.github/workflows/workload_checks.yml

```
@@ -0,0 +1,209 @@
+# Workload Checks Suite
+#
+# Runs Vector Workload Checks.
+#
+# Runs on:
+#  - scheduled daily UTC midnight
+
+# This workflow runs the collection of our workload checks, using the latest Vector nightly image,
+# which depends on when the workflow is invoked.
+#
+# The goal is to establish a baseline of check results for a variety of cases
+# and visualize trends for important Vector use cases.
+
+name: Workload Checks Suite
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
```

Teeeechnically this is off from the cycle of the nightly workflow, which triggers at 4 UTC

```suggestion
    - cron: '0 4 * * *'
```